front/pages/w/[wId]/builder/data-sources/managed.tsx

```diff
@@ -112,9 +112,11 @@ export async function setupConnection({
 
   if (
     isOAuthProvider(provider) &&
-    // Behind flag oauth-ready providers
-    // owner.flags.includes("test_oauth_setup") &&
-    ["github"].includes(provider)
+    // `oauth`-ready providers
+    (["github"].includes(provider) ||
```

Comment:
maybe just have one single array.includes ?

Reply:
Hmm these are not the same. One is the gated ones, the other the live ones

Reply:
mb indeed